specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-09-01-preview/types/experiments.json

```diff
@@ -0,0 +1,867 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Experiment Types",
+    "version": "2023-09-01-preview"
+  },
+  "paths": {},
+  "parameters": {
+    "experimentName": {
+      "name": "experimentName",
+      "description": "String that represents a Experiment resource name.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "minLength": 1,
+      "pattern": "^[^<>%&:?#/\\\\]+$",
+      "x-ms-parameter-location": "method"
+    },
+    "statusId": {
+      "name": "statusId",
+      "description": "GUID that represents a Experiment status.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "x-ms-parameter-location": "method"
+    },
+    "runningFilter": {
+      "name": "running",
+      "description": "Optional value that indicates whether to filter results based on if the Experiment is currently running. If null, then the results will not be filtered.",
+      "type": "boolean",
+      "in": "query",
+      "x-ms-parameter-location": "method",
+      "x-nullable": false
+    },
+    "executionDetailsId": {
+      "name": "executionDetailsId",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method",
+      "description": "GUID that represents a Experiment execution detail.",
+      "type": "string",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    }
+  },
+  "definitions": {
+    "experiment": {
+      "type": "object",
+      "description": "Model that represents a Experiment resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "systemData": {
+          "description": "The system metadata of the experiment resource.",
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        },
+        "identity": {
+          "description": "The identity of the experiment resource.",
+          "$ref": "./common.json#/definitions/resourceIdentity"
+        },
+        "provisioningState": {
+          "type": "string",
```

Comment:
>"type": "string", [](http://example.com/codeflow?start=10&length=17)

ProvisioningState should be an enum with at-least three values as "Succeeded", "Failed", "Canceled".